__tests__/app.test.js

```diff
@@ -78,3 +78,38 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("GET /api/articles", () => {
```

Comment:
make sure to remove .only before pushing - we can't be sure that the rest of your tests pass if there are .onlys in your code and future code will not be reviewed if we cant be sure.